src/profiler.h

```diff
@@ -40,8 +39,15 @@ static inline int cmp64(u64 a, u64 b) {
 }
 
 
+union CallTraceBuffer {
+    ASGCT_CallFrame _asgct_frames[MAX_STACK_FRAMES];
+    jvmtiFrameInfo _jvmti_frames[MAX_STACK_FRAMES];
```

Comment:
It's not used.
Is it stub  for https://github.com/jvm-profiling-tools/async-profiler/issues/51?

Reply:
Exactly